### ISSUE TITLE
[EoSD] Meiling third non fake safespot

### ISSUE THREAD

**Issue by KirbyComment** [touhouworldcup/thprac#183]
In thprac the Meiling third non creates a headsit safespot: 
https://twitter.com/iQ58sP8dNc3zKwK/status/1821183422720569771

But this safespot doesn't exist when doing full runs without thprac: 
https://twitter.com/iQ58sP8dNc3zKwK/status/1821095876011413862
https://twitter.com/iQ58sP8dNc3zKwK/status/1821113478255403489

**Comment by H-J-Granger** [comment]
I also found a bug probably related to this issue: When opening thprac and being hit by bullets near this fake safespot, you won't hear a dying sound effect. The bullets colliding with the player's hitbox will be erased as normal, and the dying animation is displayed properly, though.

**Comment by H-J-Granger** [comment]
After some practice, I found that Meiling's hitbox seems to have an offset when applying thprac to EoSD. There's a similar feedback in #27, but in IN.

I also found there are three height thresholds above Meiling's head, shown in the figure below: 

![Image](https://github.com/user-attachments/assets/822ab901-0036-4407-b88f-f5f02be955a7)

The video below shows all four circumstances:

<https://www.dropbox.com/scl/fi/nbg7wjmpqdzsyadb3zerv/Meiling-3rd-non-fake-safespot.mp4?rlkey=uk5p6bv2kjvdbiut7dufbyai5&st=o44l8ass&dl=0>

**Comment by H-J-Granger** [comment]
This fake safe spot might be very helpful to Marisa in Lunatic, since it's hard for Marisa to get through these kunais in Lunatic mode.

**Comment by H-J-Granger** [comment]
Yeah, I tested some runs warping to *Illusion Sign "Imaginary Flower Yumekazura"*, and the fake safespot seems to disappear, or at least to be smaller.
